# Project 1 -- Aayushi Akhouri

## Question 1

In [1]:
list.files("/depot/datamine/data/olympics")

[1] "athlete_events.csv" "regions.csv"

In [3]:
file.info("/depot/datamine/data/olympics/athelete_events.csv")

size isdir mode mtime ctime
/depot/datamine/data/olympics/athelete_events.csv NA   NA    <NA> <NA>  <NA> 
                                                  atime uid gid uname grname
/depot/datamine/data/olympics/athelete_events.csv <NA>  NA  NA  NA    NA

In [4]:
file.info("/depot/datamine/data/olympics/regions.csv")

size isdir mode mtime              
/depot/datamine/data/olympics/regions.csv 3595 FALSE 664  2021-08-05 17:47:40
                                          ctime              
/depot/datamine/data/olympics/regions.csv 2021-08-05 17:47:40
                                          atime               uid    gid  
/depot/datamine/data/olympics/regions.csv 2021-09-08 13:54:34 209239 20118
                                          uname   grname   
/depot/datamine/data/olympics/regions.csv kamstut tdm-admin

Markdown notes and sentences and analysis written here.

In [5]:
dim(read.csv("/depot/datamine/data/olympics/regions.csv"))

[1] 230   3

In [5]:
olympics <- read.csv("/depot/datamine/data/olympics/athlete_events.csv")

In [6]:
dim(olympics)

[1] 271116     15

In [7]:
head(olympics)

ID Name                     Sex Age Height Weight Team           NOC
1 1  A Dijiang                M   24  180    80     China          CHN
2 2  A Lamusi                 M   23  170    60     China          CHN
3 3  Gunnar Nielsen Aaby      M   24   NA    NA     Denmark        DEN
4 4  Edgar Lindenau Aabye     M   34   NA    NA     Denmark/Sweden DEN
5 5  Christine Jacoba Aaftink F   21  185    82     Netherlands    NED
6 5  Christine Jacoba Aaftink F   21  185    82     Netherlands    NED
  Games       Year Season City      Sport        
1 1992 Summer 1992 Summer Barcelona Basketball   
2 2012 Summer 2012 Summer London    Judo         
3 1920 Summer 1920 Summer Antwerpen Football     
4 1900 Summer 1900 Summer Paris     Tug-Of-War   
5 1988 Winter 1988 Winter Calgary   Speed Skating
6 1988 Winter 1988 Winter Calgary   Speed Skating
  Event                              Medal
1 Basketball Men's Basketball        NA   
2 Judo Men's Extra-Lightweight       NA   
3 Football Men's Football            NA   
4 Tug-Of-War Men's Tug-Of-War        Gold 
5 Speed Skating Women's 500 metres   NA   
6 Speed Skating Women's 1,000 metres NA

In [ ]:
#The above data set gives information about athletes in olympics.
#It displays personal and professional data of the athletes.
#It has 6 rows and 15 columns.

## Question 2

In [5]:
olympics <- read.csv("/depot/datamine/data/olympics/athlete_events.csv")

In [8]:
length(olympics$Sport)

[1] 271116

In [10]:
table(olympics$Sport)


              Aeronautics             Alpine Skiing                  Alpinism 
                        1                      8829                        25 
                  Archery          Art Competitions                 Athletics 
                     2334                      3578                     38624 
                Badminton                  Baseball                Basketball 
                     1457                       894                      4536 
            Basque Pelota          Beach Volleyball                  Biathlon 
                        2                       564                      4893 
                Bobsleigh                    Boxing                  Canoeing 
                     3058                      6047                      6171 
                  Cricket                   Croquet      Cross Country Skiing 
                       24                        19                      9133 
                  Curling                   Cycling

Markdown notes and sentences and analysis written here.

In [11]:
length(table(olympics$Sport))

[1] 66

In [13]:
unique(olympics$Sport)

[1] "Basketball"                "Judo"                     
 [3] "Football"                  "Tug-Of-War"               
 [5] "Speed Skating"             "Cross Country Skiing"     
 [7] "Athletics"                 "Ice Hockey"               
 [9] "Swimming"                  "Badminton"                
[11] "Sailing"                   "Biathlon"                 
[13] "Gymnastics"                "Art Competitions"         
[15] "Alpine Skiing"             "Handball"                 
[17] "Weightlifting"             "Wrestling"                
[19] "Luge"                      "Water Polo"               
[21] "Hockey"                    "Rowing"                   
[23] "Bobsleigh"                 "Fencing"                  
[25] "Equestrianism"             "Shooting"                 
[27] "Boxing"                    "Taekwondo"                
[29] "Cycling"                   "Diving"                   
[31] "Canoeing"                  "Tennis"                   
[33] "Modern Pentathlon"         "Figure Skating"           
[35] "Golf"                      "Softball"                 
[37] "Archery"                   "Volleyball"               
[39] "Synchronized Swimming"     "Table Tennis"             
[41] "Nordic Combined"           "Baseball"                 
[43] "Rhythmic Gymnastics"       "Freestyle Skiing"         
[45] "Rugby Sevens"              "Trampolining"             
[47] "Beach Volleyball"          "Triathlon"                
[49] "Ski Jumping"               "Curling"                  
[51] "Snowboarding"              "Rugby"                    
[53] "Short Track Speed Skating" "Skeleton"                 
[55] "Lacrosse"                  "Polo"                     
[57] "Cricket"                   "Racquets"                 
[59] "Motorboating"              "Military Ski Patrol"      
[61] "Croquet"                   "Jeu De Paume"             
[63] "Roque"                     "Alpinism"                 
[65] "Basque Pelota"             "Aeronautics"

In [15]:
length(unique(olympics$Sport))

[1] 66

In [ ]:
#I did not excpect croquet to be in the list as I had never heard of it in the olympics

## Question 3

In [3]:
olympics <- read.csv("/depot/datamine/data/olympics/athlete_events.csv")

In [13]:
us_athletes <- subset(olympics, NOC == "USA")

In [14]:
dim(us_athletes)

[1] 18853    15

In [20]:
ind_athletes <- subset(olympics, NOC == "IND")

In [21]:
dim(ind_athletes)

[1] 1408   15

In [24]:
usind_athletes <- subset(olympics,(NOC=="USA")| (olympics=="IND"))

In [25]:
dim(usind_athletes)

[1] 284203     15

There are 18853 rows in us_athletes
There are 1408 rows in ind_athletes
There are 284203 rows in usind_athletes

## Question 4

In [26]:
olympics <- read.csv("/depot/datamine/data/olympics/athlete_events.csv")

In [27]:
prop.table(table(olympics$Sex))


        F         M 
0.2748713 0.7251287 

In [34]:
prop.table(table(olympics$NOC[olympics$Sex =="F"]))


         AFG          AHO          ALB          ALG          AND          ANG 
6.709428e-05 1.610263e-04 3.623091e-04 1.261372e-03 5.635920e-04 1.731032e-03 
         ANT          ANZ          ARG          ARM          ARU          ASA 
4.964977e-04 2.683771e-05 8.628325e-03 5.099165e-04 2.147017e-04 9.393199e-05 
         AUS          AUT          AZE          BAH          BAN          BAR 
3.688844e-02 1.476074e-02 9.527388e-04 1.341886e-03 1.744451e-04 5.233354e-04 
         BDI          BEL          BEN          BER          BHU          BIH 
1.878640e-04 8.104989e-03 1.878640e-04 5.501731e-04 2.281206e-04 5.233354e-04 
         BIZ          BLR          BOH          BOL          BOT          BRA 
1.207697e-04 1.065457e-02 2.683771e-05 4.830788e-04 9.393199e-05 1.540485e-02 
         BRN          BRU          BUL          BUR          CAF          CAM 
4.159845e-04 2.683771e-05 1.434476e-02 2.012828e-04 2.147017e-04 2.281206e-04 
         CAN          CAY          CGO          CHA

7.4% of US atheletes were women
0.3% of IND atheletes were women

## Question 5

In [35]:
olympics <- read.csv("/depot/datamine/data/olympics/athlete_events.csv")

In [37]:
us_athletes <- subset(olympics, NOC == "USA")

In [46]:
subset(us_athletes,(us_athletes$Age =="75"))

ID   Name               Sex Age Height Weight Team          NOC
7434 4160 Robert Day Andrews M   75  NA     NA     United States USA
     Games       Year Season City        Sport           
7434 1932 Summer 1932 Summer Los Angeles Art Competitions
     Event                                              Medal
7434 Art Competitions Mixed Architecture, Unknown Event NA

The oldest athlete is 75 years old and he competed in art competitions in 1932

In [48]:
ind_athletes <- subset(olympics, NOC == "IND")

In [62]:
subset(ind_athletes,(ind_athletes$Age =="46"))

ID    Name                 Sex Age Height Weight Team  NOC Games      
131390 66129 Indrajit Singh Lamba M   46  176    70     India IND 1996 Summer
       Year Season City    Sport        
131390 1996 Summer Atlanta Equestrianism
       Event                                           Medal
131390 Equestrianism Mixed Three-Day Event, Individual NA

## Pledge

By submitting this work I hereby pledge that this is my own, personal work. I've acknowledged in the designated place at the top of this file all sources that I used to complete said work, including but not limited to: online resources, books, and electronic communications. I've noted all collaboration with fellow students and/or TA's. I did not copy or plagiarize another's work.

> As a Boilermaker pursuing academic excellence, I pledge to be honest and true in all that I do. Accountable together – We are Purdue.